cobbler/settings.py

```diff
@@ -141,52 +147,15 @@
     ["value", "", "", "Value", True, "Ex: 127.0.0.1", 0, "str"],
 ]
 
-if os.path.exists("/srv/www/"):
```

Comment:
Was that the part that is checked in `cobbler check`?

Reply:
Nope this is just setting the default in case there is nothing in the settings. Since the very moment we are using this the admin is configuring this, changing the default seems pretty pointless to me.